### PR TITLE
fix(#2272): pp form stepper, allow final step excluded from progress …

### PR DESCRIPTION
**Pull request by vanessatran-ddi**
Allow pp team to have the hidden 5th step as the Confirmation message 

Hidden step: https://github.com/vanessatran-ddi/latest-react-demo/blob/privacy-portal-components/src/components/2175/form-stepper.tsx#L52

How it is like: 
https://vanessatran-ddi.github.io/latest-react-demo/

### CODE REVIEW

**Review by syedszeeshan — DISMISSED** [review]

File: libs/web-components/src/experimental/ExperimentalFormStep.svelte [hunk 1]
```diff
@@ -183,20 +180,20 @@
...
-
-  label:not([aria-disabled="true"]):not([aria-current="step"]):hover {
-    background-color: rgba(0, 0, 0, 0.05);
-    cursor: pointer;
-  }
-  label.mobile:not([aria-disabled="true"]):not([aria-current="step"]):hover {
-    background-color: rgb(232 232 232);
-    cursor: pointer;
-  }
+  /*label:not([aria-disabled="true"]):not([aria-current="step"]):has(*/
```
Comment (ArakTaiRoth) [inline libs/web-components/src/experimental/ExperimentalFormStep.svelte:183]: If we don't need this stuff, maybe remove it instead of commenting it out?

**Review by syedszeeshan — APPROVED** [review]

**Pull request merged**